src/ert/config/ert_config.py

```diff
@@ -298,6 +298,8 @@ def read_templates(config_dict) -> list[tuple[str, str]]:
         templates.append([source_file, target_file])
 
     for template in config_dict.get(ConfigKeys.RUN_TEMPLATE, []):
+        if template[1].startswith("<ECL_BASE>"):
+            ConfigWarning.warn("Please use <ECLBASE> instead of <ECL_BASE>.")
```

Comment:
If these keywords are used similarly and one is deprecated, perhaps we could emphasise that?
`Keyword <ECL_BASE> is deprecated and will be removed in the future. Please use <ECLBASE> instead.`